app/config/AppConfig.scala

```diff
@@ -51,7 +51,9 @@ class AppConfig @Inject() (override val runModeConfiguration: Configuration, val
   lazy val customsDeclarationsUrl = loadConfig("urls.customsDeclarations")
 
   lazy val customsDeclarationsEndpoint = baseUrl("customs-declarations")
-  lazy val submitImportDeclarationUri = getConfString("customs-declarations.submit-uri",
+  lazy val submitExportDeclarationUri = getConfString("customs-declarations.submit-uri",
+    throw new IllegalStateException("Missing configuration for Customs Declarations submission URI"))
+  lazy val submitCancellationUri = getConfString("customs-declarations.cancel-uri",
     throw new IllegalStateException("Missing configuration for Customs Declarations submission URI"))
```

Comment:
Change an exception to:
`"Missing configuration for Customs Declarations cancellation URI"`